huaweicloud/services/dli/resource_huaweicloud_dli_queue.go

```diff
@@ -133,13 +140,6 @@ func ResourceDliQueue() *schema.Resource {
 				ForceNew:   true,
 				Deprecated: "subnet_cidr is Deprecated",
 			},
-
-			"vpc_cidr": {
```

Comment:
**Incompatible changes.**
This resource was released in V1.18.0. After about a year, this argument was deprecated in V1.26.1.

Deleting this argument will cause errors in scripts that have already been used.

V1.18.0 was released on 2020-08-29.
V1.26.1 was released on 2021-07-23.